datasetinsights/stats/image_analysis/laplacian.py

```diff
@@ -0,0 +1,85 @@
+from typing import Dict, List, Tuple
+
+import cv2
+import numpy as np
+
+
+def laplacian_img(img_path: str) -> np.ndarray:
+    """
+    Converts image to grayscale, computes laplacian and returns it.
+    Args:
+        img_path (str): Path of image
+
+    Returns:
+        np.ndarray: numpy array of Laplacian of the image
+    """
+    image = cv2.imread(img_path)
+    gray = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY)
+    laplacian = cv2.Laplacian(gray, cv2.CV_64F)
+    laplacian = laplacian.astype("float")
+    return laplacian
+
+
+def get_img_var_laplacian(img_path: str) -> np.ndarray:
+    """
+    Computes laplacian and returns the focus measure(ie variance for the image)
+    Args:
+        img_path (str): Path of image
+
+    Returns:
+        Variance of Laplacian of image
+    """
+    return laplacian_img(img_path).var()
+
+
+def get_bbox_var_laplacian(
+    laplacian: np.ndarray, x: float, y: float, w: float, h: float
```

Comment:
why set `x y w h` as `float` here if we need to transfer them as `int` in the function?

Reply:
Fixed